routes/chat.js

```diff
@@ -6,16 +6,27 @@ const chatController = require('../controllers/chat/ChatController');
 const auth = require('../middlewares/auth');
 const isTargetUserAllowingAnonDMMiddleware = require('../middlewares/chat/isTargetUserAllowingAnonDMMiddleware');
 const SuccessMiddleware = require('../middlewares/success');
-const sendSignedMessage = require('../controllers/chat/sendSignedMessage');
-const setSignedChannelAsRead = require('../controllers/chat/setSignedChannelAsRead');
 const BodyValidationMiddleware = require('../middlewares/body-validation');
 const getSignedChannelList = require('../controllers/chat/getSignedChannelList');
+const {
+  sendFollowSystemMessage,
+  setSignedChannelAsRead,
+  sendSignedMessage
+} = require('../controllers/chat');
+const {ChatValidation} = require('../joi-validations/chat.validation');
+const {validate} = require('../middlewares/joi-validation/validate');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The refactoring of the imports and function calls from `../controllers/chat` is a good improvement for readability and maintainability. It groups related functions together and makes it easier to see at a glance what functions are being used from that module.
```diff
- const sendSignedMessage = require('../controllers/chat/sendSignedMessage');
- const setSignedChannelAsRead = require('../controllers/chat/setSignedChannelAsRead');
+ const {
+   sendFollowSystemMessage,
+   setSignedChannelAsRead,
+   sendSignedMessage
+ } = require('../controllers/chat');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->